docs/constants/navigation.js

```diff
@@ -564,6 +564,7 @@ export const eas = [
     makePage('distribution/app-size.mdx'),
   ]),
   makeSection('Reference', [
+    makePage('eas/cli.mdx'),
```

Comment:
What do you think about putting this near the top? It feels like key info to me.

<img width="439" height="462" alt="Image" src="https://github.com/user-attachments/assets/26875463-8e2e-422b-938a-d23e2b8f608e" />

Reply:
Done: 
<img width="4112" height="1930" alt="CleanShot 2025-12-30 at 21 12 32@2x" src="https://github.com/user-attachments/assets/3491de43-d4d2-446b-9970-c5daeee9cc8b" />